doc: Added info about Thread child timeout's impact (#19121)
nrfconnect/sdk-nrf: doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

To avoid this issue, set the :kconfig:option:`CONFIG_CHIP_ICD_SLOW_POLL_INTERVAL` and :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL` Kconfig options to the same value (for example, ``500``).
The typical use case that the SSED is best suited for is battery-powered devices that require short response time, such as door locks or window blinds.

Child timeouts configuration
----------------------------

The device working in a Thread child role uses additional mechanisms for periodically ensuring that the communication with the parent is still possible.
These mechanisms lead to waking up the device and exchanging the messages with the parent, if the related timeout expires.

> In case of using the SED poll period value greater than any of these timeouts, the device is going to wake up more often, than it is defined by the poll period.

Reviewer: ```suggestion
In case of using the SED poll period value greater than any of these timeouts, the device wakes up more often than what is defined by the poll period.
```

> To ensure that the SED device is going to wake up exactly at every poll period value, set the following Kconfig options to the value greater than poll period value (for Matter ICD :kconfig:option:`CONFIG_CHIP_ICD_SLOW_POLL_INTERVAL`):

Reviewer: ```suggestion
To ensure that the SED device wakes up exactly at every poll period, set the following Kconfig options to the value greater than the poll period value (for Matter ICD :kconfig:option:`CONFIG_CHIP_ICD_SLOW_POLL_INTERVAL`):
```
